CARTOGRAPHY.md

```diff
@@ -37,3 +37,51 @@ account.
 * **A rich map** - This style deliberately creating a fairly rich map showing a significant number of different features.  This way it shows the richness of OSM data and gives a broad recognition to the mappers' work.  The aim is not however to show all or even most of the OSM data.
 * **Maintainability** - The implementation of this style should not be too hard to maintain. This refers to both the volume and complexity of the code and how fast the style can be parsed when rendering it, which is very important for efficient development work. So the amount of code should be kept small and complex and fragile interdependencies should be avoided. If the code is difficult to maintain this would ultimately seriously affect all of the above goals.
 * **Adaptability and ease of use** - The style should be easy to customize, like for creating localized or special purpose maps. It is also important to keep demands on rendering infrastructure for serving the style low so it is not too difficult and costly to set up a tile server for this style or a specialized variant of it.
+
+## Design guidelines
+
+The following guidelines have been found to be useful design principles that can help creating a better map in line with the above goals.  Keep in mind that none of these principles represents a goal on its own and most of them are highly condensed summaries of more complex considerations.
+
+* The difference in rendering between different types of features should foremost be based on their difference in meaning and purpose for the target map users.  Differences in physical appearance are only relevant as far as they also represent differences in meaning and purpose.
+
+* Design of symbols, area and line patterns should be in a way that avoids their geometries to be mistaken as actual data.
+
+* The connection between data and appearance of features in the map needs to be comprehensible for the viewer.
+
+* When changes have a strong impact on the overall appearance of the map input should be sought from the broader OpenStreetMap community and particular consideration given to their comments and concerns.
+
+### Colors
+
+* Differentiating features through different colors only makes sense if this difference can actually be read by the map users.
+
+* For area colors we prefer light and low saturation tones.  Lines, individual symbols and labels also can be stronger in color.
+
+* Stronger colors indicate fairly distinct and meaningful features as opposed to more general and less distinct features which use weaker colors.
+
+* Small areas work well with somewhat stronger colors while large areas tend to call for weaker tones.
```

Comment:
This is a fairly widely accepted principle but it is usually understood to be not the only guiding principle - see for example the point immediately before in the text.

If you want to remove this point i would like to see cases where you think violating it is desirable even though there are no other principles that suggest this.

Please keep in mind we do not color areas individually based on their size so this guideline refers to the typical size of areas of a certain type of course.